pennylane_lightning_kokkos/src/algorithms/AdjointDiffKokkos.hpp

```diff
@@ -279,8 +254,9 @@ template <class T = double> class AdjointJacobianKokkos {
      * @param state Statevector to be updated.
      * @param observable Observable to apply.
      */
-    inline void applyObservable(StateVectorKokkos<T> &state,
-                                const ObservableKokkos<T> &observable) {
+    inline void
+    applyObservable(StateVectorKokkos<T> &state,
+                    const Simulators::ObservableKokkos<T> &observable) {
```

Comment:
Since you import the `Simulators` namespace in the anon namepsace earlier, you should be able to ignore the `Simulators::` here and below.